torch/_inductor/cpu_vec_isa.py

```diff
@@ -161,6 +161,24 @@ def __str__(self) -> str:
     __hash__: Callable[[VecISA], Any] = VecISA.__hash__
 
 
+@dataclasses.dataclass
+class VecSVE(VecISA):
+    # this function can be repurposed for SVE with variable vec length
+    _bit_width = 256
+    _macro = [
+        "CPU_CAPABILITY_SVE",
+        "CPU_CAPABILITY_SVE256",
```

Comment:
can you point me to a logic which checks current svl == 256 at runtime? Thanks. IIRC we have it in DispatchStub.cpp for eager.